hack/v1migration/README.md

```diff
@@ -18,3 +18,15 @@
   1. Run `./hack/v1migration/restore.sh [workdir]` to restore all data from disk.
   1. Run `./hack/v1migration/scaleup.sh hive-operator` to scale up Hive operator.
   1. Run `oc patch hiveconfig hive --type='merge' -p $'spec:\n maintenanceMode: false'`
+
+
+# Resolving Problems during Restore
+
+If there was a problem during restore, then we should delete the resources in that namesapce
+that were restored, fix the data being restored, and run the restore again for just that namespace.
+
+  1. `./hack/v1migration/hive_resources_in_namespace.sh [workdir] [namespace] > [namespace_workdir]/resources.json`
+  1. `./hack/v1migration/hive_owner_refs_in_namespace.sh [workdir] [namespace] > [namespace_workdir]/owner-ref.json`
```

Comment:
If we hit something like a validation error during restore, no resource created at all, we'd want to manually setup a restore dir correct? I almost wonder if we want a script that scans the global backup dir and saved out the per namespace resources, rather than pulling them from the namespace we failed to restore.